example-wasm/src/hooks/useAppManager.ts

```diff
@@ -0,0 +1,23 @@
+import { useCallback } from 'react';
+import Realm from 'realm';
+
+const { useApp } = await import('@realm/react');
+
+/**
+ * Manages authenticating with an Atlas App.
+ */
+export function useAppManager() {
```

Comment:
Should this hook be generalised and added to `@realm/react`?

Reply:
It will be provided via the `useAuth()` and `useEmailPasswordAuth()` hooks 🙂

Reply:
It will be provided via the `useAuth()` and `useEmailPasswordAuth()` hooks 🙂